Makefile

```diff
@@ -273,41 +275,46 @@ e2e-test: e2e-bootstrap # run test for windows
 e2e-teardown: $(HELM)
 	helm delete csi-secrets-store --namespace default
 
-.PHONY: install-driver
-install-driver:
-ifdef TEST_WINDOWS
-		helm install csi-secrets-store manifest_staging/charts/secrets-store-csi-driver --namespace default --wait --timeout=15m -v=5 --debug \
-			--set windows.image.pullPolicy="IfNotPresent" \
-			--set windows.image.repository=$(REGISTRY)/$(IMAGE_NAME) \
-			--set windows.image.tag=$(IMAGE_VERSION) \
-			--set windows.enabled=true \
-			--set linux.enabled=false \
-			--set enableSecretRotation=true \
-			--set rotationPollInterval=30s
-else
-		helm install csi-secrets-store manifest_staging/charts/secrets-store-csi-driver --namespace default --wait --timeout=15m -v=5 --debug \
-			--set linux.image.pullPolicy="IfNotPresent" \
-			--set linux.image.repository=$(REGISTRY)/$(IMAGE_NAME) \
-			--set linux.image.tag=$(IMAGE_VERSION) \
-			--set linux.image.pullPolicy="IfNotPresent" \
-			--set enableSecretRotation=true \
-			--set rotationPollInterval=30s
-endif
+.PHONY: e2e-helm-deploy
+e2e-helm-deploy:
+	helm install csi-secrets-store manifest_staging/charts/secrets-store-csi-driver --namespace default --wait --timeout=15m -v=5 --debug \
+		--set linux.image.pullPolicy="IfNotPresent" \
+		--set windows.image.pullPolicy="IfNotPresent" \
+		--set linux.image.repository=$(REGISTRY)/$(IMAGE_NAME) \
+		--set linux.image.tag=$(IMAGE_VERSION) \
+		--set windows.image.repository=$(REGISTRY)/$(IMAGE_NAME) \
+		--set windows.image.tag=$(IMAGE_VERSION) \
+		--set windows.enabled=true \
+		--set linux.enabled=true \
+		--set enableSecretRotation=true \
+		--set rotationPollInterval=30s
+
+.PHONY: e2e-helm-deploy-release # test helm package for the release
+e2e-helm-deploy-release:
+	set -x; \
+	current_release=$(shell (echo ${IMAGE_VERSION} | sed s/"v"//)); \
+	helm install csi charts/secrets-store-csi-driver-$${current_release}.tgz --namespace default --wait --timeout=15m -v=5 --debug \
+		--set linux.image.pullPolicy="IfNotPresent" \
+		--set windows.image.pullPolicy="IfNotPresent" \
+		--set windows.enabled=true \
+		--set linux.enabled=true \
+		--set enableSecretRotation=true \
+		--set rotationPollInterval=30s
 
 .PHONY: e2e-kind-cleanup
 e2e-kind-cleanup:
 	kind delete cluster --name kind
 
 .PHONY: e2e-azure
-e2e-azure: $(AZURE_CLI) install-driver
+e2e-azure: $(AZURE_CLI) e2e-helm-deploy
```

Comment:
We should remove `e2e-helm-deploy` from here after https://github.com/kubernetes/test-infra/pull/21489 is merged

Reply:
Now that test-infra PR is merged removed `e2e-helm-deploy` from the `e2e-*` targets.